src/058/p058.clj

```diff
@@ -0,0 +1,37 @@
+(defn prime? [x]
+  (case x
+    1 false
+    2 true
+    (every?
+      #(not (= (rem x %) 0))
+      (range 2 (inc (Math/floor (Math/sqrt x)))))))
+
+(defn spiral-corners [side]
+  (cond
```

Comment:
`cond` is more "schemey", but one could use `case` (which is faster) or just use `if` (since it's just branching once)